backend-api/app/routes/consumers.py

```diff
@@ -44,3 +45,4 @@ async def delete_consumer(
     user: User = Depends(get_current_active_user),
 ):
     await ops.delete_consumer(db, user)
+    await user_ops.delete_user(db, user)
```

Comment:
I added a manual delete of the associated "parent" user.

Reply:
Like the comment above, we can use the get_user_manager() dependency to get a UserManager instance and delete the user.